routes.js

```diff
@@ -0,0 +1,64 @@
+const Todo = require('./models/todo');
+
+module.exports = (server) => {
+  function todoNullCheck(todo, res, id) {
+    let a = true;
```

Comment:
omg... well I'll be damned. I had it originally as `if (...){return false}{else{return true}` and the lint complained. I realized I could just do the if though.. woops